metis-validation/metis-validation-rest/src/main/resources/schema/RDF.xsd

```diff
@@ -183,4 +183,73 @@
 			</extension>
 		</simpleContent>
 	</complexType>
-</schema>
\ No newline at end of file
+
+	<complexType name="RightsStatements">
+		<attribute name="resource" use="required" form="qualified">
+			<simpleType>
+				<restriction base="string">
+					<!-- 
+					CC VERSION 1.0 LICENSES
+					<http://creativecommons.org/licenses/[PERMISSIONS]/1.0/[PORT]/>
+					PERMISSIONS: by | by-sa | by-nd | by-nc | by-nc-sa | by-nc-nd | by-nd-nc 
+					PORT: generic (no port) | fi | il | nl
+					-->
+					<pattern value="http[:]//creativecommons[.]org/licenses/(by|by-sa|by-nd|by-nc|by-nc-sa|by-nc-nd|by-nd-nc)/1[.]0/(fi|il|nl)?"/>
```

Comment:
Why are the : and . in a character class? You can just use it if it's one character.
Right?
Also some of the characters might need escaping but not sure, have you tested?

I see one issue but not sure if it's really valid:
This matches -> http://creativecommons.org/licenses/by/1.0/
This does NOT match -> http://creativecommons.org/licenses/by/1.0
Is that expected?

(optionally)The first grouping could be simplified:
(by|by-(sa|nd|nc|nc-sa|nc-nd|nd-nc))

Same applies for all other patterns in the file, which I didn't individually comment.

Reply:
the : and . need to be escaped.
The / is mandatory in the licenses, see Masa's document
about the simplification.... it is preferred to retain readability and matching what is specified in the document than optimization. You probably are aware that the automata generated from the regex is optimized internally so dont bother.

Reply:
Alright for the /.
What about the characters : .
Should I perform the change or are you going to?
I mean the removal of square brackets and escaping the .
I don't think the : requires escaping tbh.
How about the simplification?

Reply:
@hugomanguinhas Pinging you on this one